app/src/main/res/navigation/splash_graph.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/splash_graph"
+    app:startDestination="@id/splashFragment">
+
+    <fragment
+        android:id="@+id/splashFragment"
+        android:name="com.internship.move.feature.splash.SplashFragment"
+        android:label="SplashFragment" >
+        <action
```

Comment:
In OnboaringFragment if you press the back button, it should close the app.
In your case, I think you'll be returned to SplashFragment.
There's a few lines of code you need here to fix this 😁 
But first, add the safe args gradle plugin I mentioned in another comment